### PR TITLE
[MIRROR] Nukie Rework Part 2: Nuclear Operative starting gear rework [MDB IGNORE]

### PR DESCRIPTION
**Pull request by SkyratBot**
Original PR: https://github.com/tgstation/tgstation/pull/68401
--------------------

## About The Pull Request

PR Dedicated to:
image
Alan Edaggers

Replaces the nuke op Makarov with the Ansem, a clandestine pistol firing 10mm rounds which do more damage. Ammo costs more.
Replaces the nuke op survival knife with the energy dagger, as well as giving it a soft light, light armor penetration and a light wound bonus.
Replaces the diamond drill in their closet with an entrenching tool, which swaps between crowbar, pickaxe and shovel modes.
Gives the nuke op survival box the syndie box design, as well as a crowbar, screwdriver and mini welder.
Removes the nuke op leader's Krav Maga gloves.
Updates the esword and edagger sprites with ones i had lying around from 2019, they are more consistent.
Moves pistol sprites a bit up to center them.
image

## Why It's Good For The Game

Nukies having some essential tools they start with is nice, especially if they took any sort of borg that needs welding.
The krav maga gloves just got overridden by MODsuits, and they need a rethink as a whole if they even fit in the nukie arsenal.
Sprites are more consistent = good
The diamond drill being a bulky ass thing made it weird to use, a smaller entrenching tool seems better and also cooler for them.
The survival knife really doesn't cut it for nukies, 15 damage with no armor penetration is gonna be very hard to use. The energy dagger's slightly higher damage, armor penetration and respectable throw damage make it a more viable option in emergency situations where you're out of ammo.
The Ansem lets us balance nukie's starting pistol separately from the Makarov, and a more darkish weapon fits them well.

## Changelog

:cl: Fikou, NecromancerAnne, Wallem
qol: nuke ops emergency box now contains a crowbar, screwdriver and mini welder
del: nuke ops leader krav maga is removed
balance: nuke op survival knife has been replaced with an energy dagger
balance: energy daggers have light armor penetration and wound bonus
qol: energy daggers now emit a soft red light
balance: nuke op makarov has been replaced by the ansem, a 10mm clandestine pistol.
add: nuke op diamond drill has been replaced with an entrenching tool, a more compact wrench/pickaxe/shovel
/:cl:

### CODE REVIEW

File: code/modules/projectiles/boxes_magazines/external/pistol.dm [hunk 1]
```diff
@@ -27,29 +48,45 @@
...
 /obj/item/ammo_box/magazine/m9mm/fire
+<<<<<<< HEAD
 //	name = "pistol magazine (9mm incendiary)"	// SKYRAT EDIT: Original
 	name = "pistol magazine (9x25mm incendiary)"// SKYRAT EDIT: Calibre rename
 	icon_state = "9x19pI"
+=======
+	name = "pistol magazine (9mm incendiary)"
+	icon_state = "9x19pI-8"
+	base_icon_state = "9x19pI"
+>>>>>>> 71835a7c1c5 (Nukie Rework Part 2: Nuclear Operative starting gear rework (#68401))
```
Comment (tf-4) [inline code/modules/projectiles/boxes_magazines/external/pistol.dm]:
```suggestion
	//name = "pistol magazine (9mm incendiary)" // SKYRAT EDIT: Original
	name = "pistol magazine (9x25mm incendiary)" // SKYRAT EDIT: Calibre rename
	icon_state = "9x19pI-8"
	base_icon_state = "9x19pI"
```

File: code/modules/projectiles/boxes_magazines/external/pistol.dm [hunk 2]
```diff
@@ -27,29 +48,45 @@
...
 /obj/item/ammo_box/magazine/m9mm/hp
+<<<<<<< HEAD
 //	name = "pistol magazine (9mm HP)"			// SKYRAT EDIT: Original
 	name = "pistol magazine (9x25mm HP)"		// SKYRAT EDIT: Calibre rename
 	icon_state = "9x19pH"
+=======
+	name = "pistol magazine (9mm HP)"
+	icon_state = "9x19pH-8"
+	base_icon_state = "9x19pH"
+>>>>>>> 71835a7c1c5 (Nukie Rework Part 2: Nuclear Operative starting gear rework (#68401))
```
Comment (tf-4) [inline code/modules/projectiles/boxes_magazines/external/pistol.dm]:
```suggestion
	//name = "pistol magazine (9mm HP)" // SKYRAT EDIT: Original
	name = "pistol magazine (9x25mm HP)" // SKYRAT EDIT: Calibre rename
	icon_state = "9x19pH-8"
	base_icon_state = "9x19pH"
```

File: code/modules/projectiles/boxes_magazines/external/pistol.dm [hunk 3]
```diff
@@ -27,29 +48,45 @@
...
 /obj/item/ammo_box/magazine/m9mm/ap
+<<<<<<< HEAD
 //	name = "pistol magazine (9mm AP)"			// SKYRAT EDIT: Original
 	name = "pistol magazine (9x25mm AP)"		// SKYRAT EDIT: Calibre rename
 	icon_state = "9x19pA"
+=======
+	name = "pistol magazine (9mm AP)"
+	icon_state = "9x19pA-8"
+	base_icon_state = "9x19pA"
+>>>>>>> 71835a7c1c5 (Nukie Rework Part 2: Nuclear Operative starting gear rework (#68401))
```
Comment (tf-4) [inline code/modules/projectiles/boxes_magazines/external/pistol.dm]:
```suggestion
	//name = "pistol magazine (9mm AP)" // SKYRAT EDIT: Original
	name = "pistol magazine (9x25mm AP)" // SKYRAT EDIT: Calibre rename
	icon_state = "9x19pA-8"
	base_icon_state = "9x19pA"
```

File: code/modules/uplink/uplink_items/dangerous.dm [hunk 4]
```diff
@@ -26,8 +26,13 @@
 			with suppressors."
 	progression_minimum = 10 MINUTES
 	item = /obj/item/gun/ballistic/automatic/pistol
+<<<<<<< HEAD
 	cost = 5 //SKYRAT EDIT: ORIGINAL COST 7
 	purchasable_from = ~UPLINK_CLOWN_OPS
+=======
+	cost = 7
+	purchasable_from = ~(UPLINK_NUKE_OPS | UPLINK_CLOWN_OPS)
+>>>>>>> 71835a7c1c5 (Nukie Rework Part 2: Nuclear Operative starting gear rework (#68401))
```
Comment (tf-4) [inline code/modules/uplink/uplink_items/dangerous.dm]:
```suggestion
	cost = 5 //SKYRAT EDIT: ORIGINAL COST 7
	purchasable_from = ~(UPLINK_NUKE_OPS | UPLINK_CLOWN_OPS)
```

File: code/modules/projectiles/boxes_magazines/_box_magazine.dm [hunk 5]
```diff
@@ -173,15 +175,19 @@
...
 			icon_state = "[initial(icon_state)]-[shells_left ? "[max_ammo]" : "0"]"
 		//SKYRAT EDIT ADDITION BEGIN - SEC_HAUL
 		if(AMMO_BOX_FULL_EMPTY_BASIC)
 			icon_state = "[initial(icon_state)]-[shells_left ? "full" : "empty"]"
 		//SKYRAT EDIT END
 	desc = "[initial(desc)] There [(shells_left == 1) ? "is" : "are"] [shells_left] shell\s left!"
 	update_custom_materials()
+=======
+			icon_state = "[multiple_sprite_use_base ? base_icon_state : initial(icon_state)]-[shells_left ? "[max_ammo]" : "0"]"
+>>>>>>> 71835a7c1c5 (Nukie Rework Part 2: Nuclear Operative starting gear rework (#68401))
```
Comment (tf-4) [inline code/modules/projectiles/boxes_magazines/_box_magazine.dm]:
```suggestion
			icon_state = "[[multiple_sprite_use_base ? base_icon_state : initial(icon_state)]-[shells_left ? "[max_ammo]" : "0"]"
		//SKYRAT EDIT ADDITION BEGIN - SEC_HAUL
		if(AMMO_BOX_FULL_EMPTY_BASIC)
			icon_state = "[[multiple_sprite_use_base ? base_icon_state : initial(icon_state)]-[shells_left ? "full" : "empty"]"
		//SKYRAT EDIT END
```

**Comment by Lukaster42** [comment]
holy shit, ugly makarov

**Pull request merged**